bin/p-s.js

```diff
@@ -55,7 +56,25 @@ runPackageScript({
 function getPSConfig() {
   var psConfigFilename = program.config ? resolve(process.cwd(), program.config) : findUp.sync('package-scripts.js')
   try {
-    return require(psConfigFilename)
+    // Require a module like babel-register
+    if (program.require) {
+      try {
+        require(program.require)
```

Comment:
If someone wanted to `--require ./some-file/in/their-project.js` this wouldn't work. I think there's a way to use `resolve` to make this work relative to `process.cwd()`.